hyco-https/tests/get.test.js

```diff
@@ -34,11 +39,11 @@ test('HTTPS GET', (done) => {
         expect(err).toBeUndefined();
     });
 
-    // client is being run with 5 second delay to allow the server to settle
```

Comment:
>// client is being run with 5 second delay to allow the server to settle [](start = 4, length = 72)

I notice this is now gone. Was it just not needed?

Reply:
It could be part of the total timeout. Removing this will allow the test to finish without waiting for this delay.

---
In reply to: [315836027](https://github.com/Azure/azure-relay-node/pull/32#discussion_r315836027) [](ancestors = 315836027)